packages/auth/src/grant/routes.ts

```diff
@@ -144,12 +157,26 @@ async function createPendingGrant(
   const { body } = ctx.request
   const { grantService, interactionService, config } = deps
   if (!body.interact) {
-    ctx.throw(400, 'interaction_required', { error: 'interaction_required' })
```

Comment:
I changed this from `interaction_required` because that seems to be more for if a resource (likely only a grant) requires interaction in order to be accessed.